iteration/iteration.go

```diff
@@ -69,6 +75,7 @@ type Repository interface {
 	CanStart(ctx context.Context, i *Iteration) (bool, error)
 	LoadMultiple(ctx context.Context, ids []uuid.UUID) ([]Iteration, error)
 	LoadChildren(ctx context.Context, parentIterationID uuid.UUID) ([]Iteration, error)
+	CalculateActiveIterationOnUpdate(ctx context.Context, itr *Iteration) (*bool, error)
```

Comment:
Why isn't this just a Method on the Iteration object?

    Iteration.IsActive()